### ISSUE TITLE
Demo recording and playback

### ISSUE THREAD

**Issue by AlexMax** [joezeng/cambridge#19]
Are there plans for being able to record and play back demos of gameplay?

**Comment by joezeng** [comment]
Yes, eventually. I haven't gotten around to it yet, but it is definitely on the roadmap.
